examples/nextjs-next/pages/_app.js

```diff
@@ -9,7 +9,7 @@ class MyApp extends App {
   componentDidMount() {
     // Remove the server-side injected CSS.
     const jssStyles = document.querySelector('#jss-server-side');
-    if (jssStyles) {
+    if (jssStyles && jssStyles.parentNode) {
```

Comment:
What case are we covering here? Seems like the element shouldn't be orphaned in the first place, no?

Reply:
Indeed, but as far as typescript can see, `.parentNode` can be `null`.

Reply:
Yes for orphaned styles indeed. But this should be considered a bug and not just silently ignored. Since this isn't a typescript file we don't need to concern ourselves with it. 

In a typescript file we would just use `jssStyles.parentNode!.foo` and wait for runtime errors.